### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by ParadauxDev** [richardbeattie/Javascipt-scroll#1]
What's the purpose of the goTo() method? HTML Anchors exist
http://geekswithblogs.net/ihaynes/archive/2013/07/12/named-anchors-in-html5.aspx

**Comment by richardbeattie** [comment]
Yeah but they don't **smooth scroll** to the desired element. Using anchors is more akin to teleporting to the element while the goTo() method will scroll down to the element you specify.
